### DISCUSSION TITLE
Freie Wahl der Tradition bei Zauber

### DISCUSSION THREAD

**Discussion by ghost** [elyukai/optolith-client#1153] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Hallo ist es möglich in einem zukünftigen Release die Tradition seiner Zauber selber auszuwählen?

**Comment by elyukai** [comment, marked as answer]
In einer zukünfigen Release bestimmt! Siehe dazu auch #28. Da Optolith primär allerdings auf strikte Regelauslegung ausgelegt ist, steht das nicht ganz oben auf der Liste. Ich kann dir daher leider keine Auskunft geben, wann das möglich sein wird. Falls es sehr dringend ist, müsstest du daher entweder auf "Platzhalter-Zauber" ausweichen oder eben andere Tools wie den Selbstrechnenden Heldenbogen verwenden, die diese Freiheit meines Wissens nach gewähren, da sie die Regeln nicht strikt kontrollieren.
